api/v1alpha1/flowcollector_types.go

```diff
@@ -214,6 +214,14 @@ type FlowCollectorEBPF struct {
 	// (for example GOGC, GOMAXPROCS env vars). Users setting its values do it at their own risk.
 	// +optional
 	Debug DebugConfig `json:"debug,omitempty"`
+
+	// `enableFlowRtt` allows enabling FlowRTT calculations in the ebpf agent.
+	// Currently only TCP handshake based RTT is calculated per flow.
+	// This feature is optionally enabled and is disabled by default.
+	// This feature needs both INGRESS and EGRESS direction flow capture and will be disabled if both are not enabled.
+	//+kubebuilder:default:=false
+	//+optional
+	EnableFlowRTT bool `json:"enableFlowRtt,omitempty"`
```

Comment:
Since it's only available for TCP and for consistency with [enableTCPDrop](https://github.com/netobserv/network-observability-operator/pull/331/files#diff-8bc0fb83e12fdfcc0c2cf841134d869ee9f619d4724a2d5c9f8919ea2823a82aR230) option what do you think about 

```suggestion
	EnableTCPRTT bool `json:"enableTCPRtt,omitempty"`
```

Reply:
Actually the problem that I see is that not making it generic will mean name change when we introduce any new update to the RTT functionality.
Infact I was mentioning this before that TCP is just one facet and we need to consider the ICMP Echo Request/Response pair as well for calculating RTT.
I had kept the ebpf agent side generic as well to enable future changes. If that is okay can we keep it generic for now?

Reply:
yes it makes sense :+1: 
@jotak WDYT ?

Reply:
sure works for me I can convert this to Enum.

Reply:
maybe just wait for other PRs to be merged, or we can do as followup :wink: